transform/macros/repo_event_by_period.sql

```diff
@@ -0,0 +1,11 @@
+{% macro repo_event_by_period(event_type, period) -%}
+select distinct
+  repo_id,
+  repo_name,
+  date_trunc('{{ period }}', event_date) as date_{{ period }},
+  count(*) as {{ event_type }}_count,
+  sum({{ event_type }}_count) over (partition by repo_id order by date_{{ period }}) as cumul_{{ event_type }}_count
+from {{ ref("stg_gharchive") }}
+where event_type = '{{ event_type | capitalize }}'
```

Comment:
Unless I'm missing something, `{{ event_type | capitalize }}` won't work since the event types in our staging layer are now lowercase? Let's make sure to test the code and check the tables are properly created and populated before re-review 🙏